### DISCUSSION TITLE
Feature rating with Francesco

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/LinkFree#9764] [General]
YouTube live https://www.youtube.com/watch?v=PodGKBbNle4

- [x] stats daily views
- [x] header/footer remove
- [x] stats for links
- [ ] *pwa mobile app*
- [ ] *follow and notifications* (mvp: email, notification centre, grouping notifications)
- [x] testimonials
- [x] stats referrers
- [x] repos
- [x] events
- [x] map
- [x] milestones
- [x] custom domain
- [x] stats for location
- [ ] *redesign (including nextjs router, typescript, prisma...)*

---

Q. magic wand... any feature?
1. social media stats
1. social post (pinned)
1. social / community
1. video testimonials
1. profile video (embedded from YouTube?)

## IDEAs

- Repos to have multiple profile pictures and clickable
- Social embedded (favourite content) + global social wall
-

**Comment by ikrishg** [comment]
1. map (because im a big fan of @amandamartin-dev)
1. custom domains (yeah... you know why) 😂 
1. header/footer remove
1. stats
1. testimonials (i really love breaking them)
1. link animations (now i wanna give everything an animation, but that much is too much)
1. repos
1. events (good, but too much effort)
1. milestones (yeah duplicating them on LinkedIn, polywork and now another platform? too much effort)

---
Needed features:

1. vercel (mission already accomplished)
1. prisma
1. typescript
1. shadcn/ui
1. nextjs app dir
1. better analytics (using tinybird db maybe)
1. algorithm for making the ranking system better
1. follow & notifications
1. ---breaking things by deploying on fridays---
1. pwa (not my cuppa tea)

---
Magic:
1. maybe, just maybe, can we have sync between our bio, avatar and username with every single social media? so that when i change my avatar i don't have to go everywhere to change it
1. blog showcase (like repo showcase)
